lib/features/payments/review_request_page.dart

```diff
@@ -0,0 +1,171 @@
+  final String outputAmount;
+  final String inputCurrency;
+  final String outputCurrency;
+  final String exchangeRate;
+  final String serviceFee;
+  final String bankName;
+  final Map<String, String> formData;
+  final String transactionType;
+
+  const ReviewRequestPage({
+    required this.inputAmount,
+    required this.outputAmount,
+    required this.inputCurrency,
+    required this.outputCurrency,
+    required this.exchangeRate,
+    required this.serviceFee,
+    required this.bankName,
+    required this.formData,
+    required this.transactionType,
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(),
+        body: SafeArea(
+            child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: Grid.side),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Expanded(
+                      child: SingleChildScrollView(
+                          physics: const BouncingScrollPhysics(),
+                          child: Column(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: [
+                                _buildHeader(context),
+                                const SizedBox(height: Grid.md),
+                                _buildAmounts(context),
+                                const SizedBox(height: Grid.md),
+                                _buildFeeDetails(context),
+                                const SizedBox(height: Grid.md),
+                                _buildBankDetails(context),
+                              ])),
+                    ),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.stretch,
+                      children: [
+                        FilledButton(
+                          onPressed: () {
+                            Navigator.of(context).push(
+                              MaterialPageRoute(
+                                builder: (context) => SuccessPage(
+                                  text: Loc.of(context).yourRequestWasSubmitted,
+                                ),
+                              ),
+                            );
+                          },
+                          child: Text(Loc.of(context).submit),
+                        ),
+                      ],
+                    ),
+                  ],
+                ))));
+  }
+
+  Widget _buildHeader(BuildContext context) {
+    return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+      Text(
+        Loc.of(context).reviewYourRequest,
+        style: Theme.of(context).textTheme.titleLarge,
+      ),
+      const SizedBox(height: Grid.xs),
+      Text(Loc.of(context).makeSureInfoIsCorrect,
+          style: Theme.of(context).textTheme.bodyLarge),
+    ]);
+  }
```

Comment:
Totally optional, but you can use a shorthand for things that just return like this. 

```dart
Widget _buildHeader(BuildContext context) => Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
      Text(
        Loc.of(context).reviewYourRequest,
        style: Theme.of(context).textTheme.titleLarge,
      ),
      const SizedBox(height: Grid.xs),
      Text(Loc.of(context).makeSureInfoIsCorrect,
          style: Theme.of(context).textTheme.bodyLarge),
    ]);
```